editor/src/components/canvas/canvas-strategies/reparent-strategy-helpers.ts

```diff
@@ -241,6 +298,19 @@ export function getReparentTargetUnified(
   )
 
   const filteredElementsUnderPoint = allElementsUnderPoint.filter((target) => {
+    let validParentForFlexOrAbsolute = cmdPressed
```

Comment:
I wonder if pressing cmd should "force" the other strategy to win instead of letting the strict strategy become permissive